packages/fxa-payments-server/public/index.html

```diff
@@ -10,6 +10,8 @@
     <meta name=robots content=noindex,nofollow>
     <meta name=viewport content="width=device-width,initial-scale=1,maximum-scale=2,user-scalable=yes">
     <meta name=apple-itunes-app content="app-id=989804926, affiliate-data=ct=smartbanner-fxa">
+    <meta name="fxa-content-server/config" content="__SERVER_CONFIG__">
+    <meta name="fxa-feature-flags" content="__FEATURE_FLAGS__">
```

Comment:
I picked arbitrary strings, I don't know if there's any special pattern. I didn't think we should use Mustache here because it's a rendered page, not a template.